### DISCUSSION TITLE
Mobile phones eligibility

### DISCUSSION THREAD

**Discussion by KajalPandey15895** [snigi-gupta/AttendingUniversityAtBuffalo#18] [Q&A] [answered]
Hi,

First of all, Great work with the detailed information for each and every topic. 

I had few query regarding Mobile phones.
I have read few articles and also heard from few seniors that not all Indian mobile phone brand work in USA because of common issue like compatibility with network provider. 
I want to change my phone because it's not working properly and it might cause me issue while travelling .
But I have received advice to buy phones in USA. That being said I know I won't be able to buy mobile phone ASAP as there will be other things which will be on priority and my phone is not in the condition to hold on till then.

So, Can I Know which brands phone will be okay for me to carry with me with respect to compatibility with network provider?
It will be great if you can suggest few names too.

Thanks in Advance.

**Comment by snatarajappa** [comment, marked as answer]
@KajalPandey15895  - As far as I know, Google Pixel, iPhone shouldn't have any issues. You can buy them if they are within your budget.

**Reply by snigi-gupta** [reply]
@KajalPandey15895 there are webiste on the internet that can tell you if your phone will be compatible in USA or not. So you can specifically check that for you your phone.

As @snatarajappa  said, yes, Pixel, iPhone, oneplus, motorola will not have any issues. 

If you arrive in USA and cannot go out and buy one immediately, you can order from Bestbuy or amazon. phones are cheaper here. if money is an issue then i would suggest buy a decent economical phone from India and use that.
